README.md

```diff
@@ -13,41 +13,39 @@
 1. [Deployment & Release](#deployment--release)
 1. [Versioning](#versioning)
 
-## Install
+## Installation
```

Comment:
You can not have two sections with the same name (the links would not work), therefore I went with `Install`.

Reply:
How about `Setup`?